includes/functions/articles.php

```diff
@@ -165,23 +167,36 @@ function get_columns() {
 	 *
 	 * Gets the columns (and sets the internal headers property)
 	 * Gets the \WP_Post for each article ID in the current object's set
-	 *
 	 */
 	function prepare_items() {
 		$columns               = $this->get_columns();
-		$hidden                = [];
-		$sortable              = [];
-		$this->_column_headers = [ $columns, $hidden, $sortable ];
+		$hidden                = array();
+		$sortable              = array();
+		$this->_column_headers = array( $columns, $hidden, $sortable );
 		foreach ( $this->article_ids as $id ) {
 			$post = get_post( $id );
+
+			$post_img_num   = 0;
+			$gallery_images = get_post_galleries( $post, true );
+			if ( ! empty( $gallery_images ) ) {
+				foreach ( $gallery_images as $single ) {
+					$post->post_content .= $single;
+				}
+			}
+			$post_img_num = (int) preg_match_all( '/<img[^>]*>/', $post->post_content, $array );
```

Comment:
**Non-Blocker:** We should name this `$array` variable as `$matches`

Reply:
@sksaju Done